### DISCUSSION TITLE
Move back to Veekun

### DISCUSSION THREAD

**Discussion by Naramsim** [PokeAPI/pokeapi#677] [General]
We should understand whether to go back to Veekun or to keep updating the data by ourselves.

@C-Garza I would like your help here if you have time and will.

Basically now our data diverges from theirs, while, they added as well gen8 data. I think their data is more in line with the games released. Now since we decided to diverge we didn't add only gen8 data to our repository but we added as well some data related to other things that veekun might lack.

I think we have two options here:

- stay diverged and let our contributors update our data on demand
- make a list of PR that changed non-gen8-related data, replace our data with veekun's, attempt to recreate those PRs (either here of maybe directly at veekun)

Merging our data with theirs and resolving git conflicts is not a viable alternative since our IDs change with their and the conflicts become hard to resolve.

**Comment by C-Garza** [comment]
Is their gen8 data much different from ours? I think the second option you list might be the way to go, or else we'll end up pulling data from veekun and the fork and it'll just get messier if they diverge more and more.

**Reply by Naramsim** [reply]
Hi @C-Garza, follows a list of bigger differences between their files and ours.

- `ability_flavor_text` they have multiple languages
- `evolution_triggers` they have more evolution triggers
- `move_effect_prose` they are incomplete
- `moves` they have same moves different values
- `pokemon_abilities` they have less abilities
- `pokemon_dex_numbers` we lack data
- `pokemon_evolutions` they have more evolutions
- `pokemon_form_names` they have translations
- `pokemon_forms` they have different values and less forms
- `pokemon_species` they have different values
- `pokemon_stats` they have different values and less stats
- `pokemon_types` same as above
- `pokemon` they have different values and less pokemon

In the end I think we can merge the two branches and fix the conflict little by little. I'll try creating a small pull request integrating their data and see how it goes.
